packages/esm-commons-lib/src/index.ts

```diff
@@ -49,3 +52,12 @@ export * from './components/expandable-list-widget/expandable-list.component';
 export * from './components/extension-conditional-renderer/patient-based-extension-renderer';
 export * from './components/patient-table/patient-table.component';
 export * from './hooks/useLastEncounter';
+
+// Workspace registration moved to the index.ts and routes.json
+const options = {
+  featureName: 'ohri-forms-workspace-item',
+  moduleName: "@ohri/openmrs-esm-ohri-commons-lib",
+};
+
+// t('ohriForms', "Ohri Forms")
```

Comment:
```suggestion
// t('ohriForms', "OHRI Forms")
```

Seems more correct to me.